recipes/bkcharts/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "bkcharts" %}
+{% set version = "0.2" %}
+{% set sha256 = "a5eaa8e78853dcecaa46345812e4fabe9cd3b96330ebf0809f640a4a0556d72e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - six >=1.5.2
+    - numpy >=1.7.1
+    # xref: https://github.com/bokeh/bkcharts/issues/72
+    - pandas
```

Comment:
`pandas` is used throughout, but is not explicitly a dependency. Have raised to upstream to see if it can be added as a dependency there.

xref: https://github.com/bokeh/bkcharts/issues/72